docs/product/issues/issue-details/index.mdx

```diff
@@ -46,14 +61,16 @@ To view other events, you can use the dropdown to skip to the latest or oldest e
 
 ![Trace Navigator](./img/trace-navigator.png)
 
-The trace navigator (which you'll see right below the Event ID), is a timeline view that shows any other issues that might be part of the current event's [trace](/product/sentry-basics/tracing/distributed-tracing). The trace navigator shows up to 100 additional issues and you can click on any of them to view their details. Seeing (and being able to toggle between) multiple issues connected to a single event can be helpful for figuring out what went wrong. Note that the trace navigator is only available for issues that are part of a trace and have more than one issue.
-
-Issues in the trace navigator timeline are represented by dots of various shades of red. Here are the different types  to note:
+The trace navigator (which displays below the date) is an abbreviated view of the related [trace](/product/sentry-basics/tracing/distributed-tracing) for the current transaction. It displays up to six nodes, each representing different groups of the event's trace:
```

Comment:
This isn't new documentation. This page had the older copy/screenshots of the Trace Navigator which were taking from `/issus/issue-details/error-issues/` and then removed from there.